Sources/TuistSupport/Constants.swift

```diff
@@ -23,6 +23,7 @@ public struct Constants {
     public struct DerivedDirectory {
         public static let name = "Derived"
         public static let infoPlists = "InfoPlists"
+        public static let sources = "Sources"
```

Comment:
@kwridan what do you think about using this name for the folder? Should we be more specific with the name?

Reply:
`Sources` looks good, it can be changed later if we start adding more sources to group / categorize